src/gurt/errno.c

```diff
@@ -133,3 +142,22 @@ d_errno_deregister_range(int start)
 	D_ERROR("Attempted to deregister non-existent error range from %d\n",
 		start);
 }
+
+const char *
+d_strerror(int errnum)
```

Comment:
silly complaint perhaps but there is some inconsistency in naming here between d_errstr and d_strerror.   Perhaps something like 

d_errstr and d_errdesc ?

Reply:
And if d_strerror() is now d_errdesc() (is it?), this should appear in commit-msg and in CART-842 JIRA ticket!

Reply:
Done